policy-controller/k8s/index/src/index.rs

```diff
@@ -609,6 +612,76 @@ impl kubert::index::IndexNamespacedResource<k8s::policy::NetworkAuthentication>
     }
 }
 
+impl kubert::index::IndexNamespacedResource<k8s_gateway_api::HttpRoute> for Index {
+    fn apply(&mut self, route: k8s_gateway_api::HttpRoute) {
+        let ns = route.namespace().expect("HttpRoute must have a namespace");
+        let name = route.name();
+        let _span = info_span!("apply", %ns, %name).entered();
+
+        let route_binding = http_route::RouteBinding::from_resource(route);
+
+        self.ns_or_default_with_reindex(ns, |ns| ns.policy.update_http_route(name, route_binding))
+    }
+
+    fn delete(&mut self, ns: String, name: String) {
+        let _span = info_span!("delete", %ns, %name).entered();
+        self.ns_with_reindex(ns, |ns| ns.policy.http_routes.remove(&name).is_some())
+    }
+
+    fn reset(
+        &mut self,
+        routes: Vec<k8s_gateway_api::HttpRoute>,
+        deleted: HashMap<String, HashSet<String>>,
+    ) {
+        let _span = info_span!("reset").entered();
```

Comment:
these spans don't indicate what resource type is being applied/deleted/reset...should they maybe include something like
```suggestion
        let _span = info_span!("apply", resource = %"HTTPRoute", %ns, %name).entered();

        let route_binding = http_route::RouteBinding::from_resource(route);

        self.ns_or_default_with_reindex(ns, |ns| ns.policy.update_http_route(name, route_binding))
    }

    fn delete(&mut self, ns: String, name: String) {
        let _span = info_span!("delete", resource = %"HTTPRoute" %ns, %name).entered();
        self.ns_with_reindex(ns, |ns| ns.policy.http_routes.remove(&name).is_some())
    }

    fn reset(
        &mut self,
        routes: Vec<k8s_gateway_api::HttpRoute>,
        deleted: HashMap<String, HashSet<String>>,
    ) {
        let _span = info_span!("reset", resource = %"HTTPRoute").entered();
```
or similar?

Reply:
pretty sure the resource type is already on a parent span.